forge-gui/res/cardsfolder/b/bloodthirsty_adversary.txt

```diff
@@ -11,4 +11,5 @@ SVar:DBCopyCast:DB$ Play | Valid$ Card.IsRemembered | ValidZone$ Exile | Control
 SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
 SVar:X:Count$TriggerRememberAmount
 DeckHas:Ability$Counters
+AI:RemoveDeck:All
```

Comment:
This one needs two thoughts for the AI:
* First check for Instance and Sorcery with cmc 3 or less that are okay with being cast for free
* Then Check if the Extra Mana to get more counters are worth it (is 1 counter for 2 R enough to be lethal?)